lib/vorpal.js

```diff
@@ -397,6 +398,11 @@ vorpal._hook = function (fn) {
 };
 
 /**
+ * History module used to get command history
+ */
+vorpal.cmdHistory = new History();
```

Comment:
This needs to be created separately upon instantiating each Vorpal instance, i.e. it should go in the `Constructor`.

Vorpal is designed to be able to run multiple instances, where only one of them is actively connected to the UI. Each instance should have its own `History` instance.